.teamcity/pluginData/settings.xml

```diff
@@ -0,0 +1,23 @@
+<settings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/SETTINGS/1.0.0" xsi:schemalocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <servers>
+        <server>
+            <id>ossrh</id>
```

Comment:
What is `ossrh` ?

Reply:
It's derived from this: https://central.sonatype.org/pages/ossrh-guide.html